Docs/user_guide/modules/markups.md

```diff
@@ -163,6 +237,24 @@ ROI settings section:
 - Display ROI: show/hide the ROI in all views.
 - Interactive mode: allow changing the ROI position, orientation, and size in views using interaction handles. If interaction handles are disabled, the ROI may still be changed by moving control points (unless control points are locked, too).
 
+## Creating Template Landmarks
+
+In order to define a workflow in which a known list of markups must be placed, it is possible to save/load predefined markups lists to use as templates.
+
+Defining a template:
+  1. Create a new point list using the toolbar, or the create button in the Markups module. You may want to exit place mode by clicking on the toolbar, or by right-clicking in a view.
+  1. In the Markups module, select the newly created points list
+  1. In the Control Points - Advanced section, create the required number of control points for the template by clicking on the Add Control Point button.
+  1. Set the name/description of the control points in the control points table.
+  1. To prevent points from being added or removed, click on the "Fixed list of points" button at the top of the control points section.
+  1. Export the template to file by right-clicking on the points list in the markups module, or in the subject hierarchy, and select "Export to file...".
+
+Loading a template:
+  1. Drag-and-drop the template file into 3D Slicer.
+  1. Select the loaded template in the markups toolbar.
+  1. Enter place mode with "Place multiple control points" enabled.
+  1. Place points until all of the points in the template have been placed. Each point in the template will be selected for placement sequentially. Once all points have been placed, place mode will be automatically disabled.
```

Comment:
Will need to update the numbered list instead of everything being `1.`

Reply:
For style reasons? Using `1.` auto increments the list: https://slicer--6643.org.readthedocs.build/en/6643/user_guide/modules/markups.html#creating-template-landmarks

Reply:
I think the rational for this was to:
* have the markdown rendering engine take care of the numbering[^1]
* minimize the amount of code change when adding a new item to the list

That said, for consistency with existing documentation, numbering makes sense as this instance would be the only occurrence. 

[^1]: https://slicer--6643.org.readthedocs.build/en/6643/user_guide/modules/markups.html#creating-template-landmarks

Reply:
I didn't know the renderer could do that automatically. That's cool 👍🏻

Reply:
@jamesobutler Yeah, it's a useful feature for lazy list making.
@jcfr Sure, makes sense.